services/traefik/dynamic_conf.yml

```diff
@@ -0,0 +1,11 @@
+tls:
+  certificates:
+  - certFile: /run/secrets/domain.crt
+    keyFile: /run/secrets/domain.key
+    stores:
+    - default
+  stores:
```

Comment:
I guess having strict sni on would be fine ... after all supporting old browsers is a security risk in itself and thus not having a defaultCertificate might be actually an advantage

Reply:
you're the expert there. so I should remove the defaultCertificate?

Reply:
done